### PR TITLE
Minor fix for AngularSelect

### PR DESCRIPTION
**Pull request by lehinevych**
Issue was, that Angular Select didn't choose the item from list, mentioned during testing the Service Dialog creation.

{{pytest: cfme/tests/infrastructure/test_providers.py cfme/tests/services/test_catalog_item.py -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 616b9f0d3f1b6d29aaae51534386c3e319bdbf8b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 807a2c754d516ae0d76e04f1995203d27b99d91d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 924a6bad6bfc48ac4425e95be3662b5e233b20a3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/catalogs/catalog_item.py [hunk 1]
```diff
@@ -34,7 +35,9 @@
         ('select_provider', Select("//select[@id='manager_id']")),
         ('field_entry_point', Input("fqname")),
         ('edit_button', form_buttons.save),
-        ('apply_btn', '//a[@title="Apply"]')
+        ('apply_btn', {
+            version.LOWEST: '//a[@title="Apply"]',
+            version.LATEST: '//a[@data-original-title="Apply"]'})
```
Comment (psav) [inline cfme/services/catalogs/catalog_item.py]: This should probably be
```python
        ('apply_btn', {
            version.LOWEST: '//a[@title="Apply"]',
            '5.5': '//a[@data-original-title="Apply"]'})
```

File: cfme/web_ui/__init__.py [hunk 2]
```diff
@@ -3287,7 +3287,7 @@ def open(self):
     def select_by_visible_text(self, text):
         if not self.is_open:
             self.open()
-        new_loc = self._loc + '/../div/ul/li[contains(., "{}")]'.format(text)
+        new_loc = self._loc + '/../div/ul/li/a[contains(., "{}")]'.format(text)
```
Comment (psav) [inline cfme/web_ui/__init__.py]: This works for all Selects?

File: cfme/services/catalogs/catalog_item.py (hunk 1 above)
Comment (lehinevych) [inline cfme/services/catalogs/catalog_item.py]: '//a[@data-original-title="Apply"]' works only for upstream.

File: cfme/web_ui/__init__.py (hunk 2 above)
Comment (lehinevych) [inline cfme/web_ui/__init__.py]: For all I tested manually and those which was tested in PRT

**Comment by dajoRH** [comment]
Lint report for commit 8a87fe9ce051e1688cd648413ea24761f6bf8c6c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dcc35b3fe969b2dd6b3255ccebbc9581add6955f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/catalogs/catalog_item.py (hunk 1 above)
Comment (psav) [inline cfme/services/catalogs/catalog_item.py]: Ahh ok, my mistake ;)

**Comment by lehinevych** [comment]
The test "test_provider_add_with_bad_credentials[rhevm34]" fails on downstream-55z  and upstream 
because of the returned message "Credential validation was not successful: wrong number of arguments (2 for 0)" doesn't match expected one "401 Unauthorized"

**Comment by dajoRH** [comment]
Lint report for commit 4ec9d36ffe932d8d173b31edc207a5ccd5d2cd9e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6f5c43e8fee42970545238a8e411179d871a95f5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**